keanu-python/keanu/vertex/base.py

```diff
@@ -22,14 +23,15 @@
 class Vertex(JavaObjectWrapper, SupportsRound['Vertex']):
 
     def __init__(self, val_or_ctor: Union[JavaMember, JavaObject],
-                 *args: Union[vertex_constructor_param_types, shape_types]) -> None:
+                 *args: Union[vertex_constructor_param_types, shape_types], **kwargs: Dict[str, Any]) -> None:
```

Comment:
Is there a reason you're using kwargs here rather than adding an explicit label=None parameter?